src/Symfony/Component/Form/Util/FormUtil.php

```diff
@@ -41,4 +43,13 @@ public static function isEmpty($data)
         // not considered to be empty, ever.
         return null === $data || '' === $data;
     }
+
+    /**
+     * Appends a block prefix after the static defaults and before dynamic ones.
+     */
+    public static function appendStaticBlockPrefix(FormView $view, string $prefix, bool $decreaseOffset): void
```

Comment:
I would not add this method. To me it decreases readability a lot to not just see in place how `array_splice()` is used.